### ISSUE TITLE
Rows should contain multiple items.

### ISSUE THREAD

**Issue by tdresser** [gizmomogwai/org-kanban#26]
Right now, each row contains a single entry.
Collapsing entries upwards would save a fair bit of space.

Here's an example of what it looks like:

```
| TODO | BLOCKED | IN_PROGRESS | DONE   |
|------+---------+-------------+--------|
| test | test 1  | test 2      | test 4 |
|      |         | test 3      | test 5 |
```

Pardon my elisp inexperience - I'm currently using:

```elisp
(defun transpose-lists (x)
  (apply '-zip-lists (apply '-pad nil x)))

(defun point-offset (point columns rows)
  (save-excursion
    (goto-char point)
    (forward-line columns)
    (forward-char rows)
    (point)))

(defun org-table-collapse (x)
  (let* ((lines (split-string x "\n"))
	 (matrix (--map (split-string it "|") lines))
	 (collapsed-matrix
	  (transpose-lists
	   (--map (--filter (not (or
				  (equal nil it)
				  (string-blank-p it))) it)
		  (transpose-lists matrix)))))
    (string-join
     (--map (string-join it "|") collapsed-matrix) "\n")))

(defun org-table-collapse-at-point ()
  (interactive)
  (unless (org-at-table-p) (user-error "Not at a table"))
  (let ((data-start (point-offset (org-table-begin) 2 0))
	(data-end (point-offset (org-table-end) 0 -1)))
    (insert (org-table-collapse 
	     (delete-and-extract-region data-start data-end)))
    (org-table-align)))

(advice-add #'org-dblock-write:kanban :after
	    (lambda (dummy) (org-table-collapse-at-point)))
```

**Comment by gizmomogwai** [comment]
mh ... its a great idea.
not sure if i will have the time to do it in the not too distant future.
may i ask how you use org-kanban? do you interact in the kanban board, or do you do this on the individual todos?
i think i could render a read only kanban board quite simply, but interacting with it would be more complicated. e.g. i would have to keep better track of the item i am on, if i move it left or right and it changes its row.

**Comment by tdresser** [comment]
The only interaction I perform is clicking on items, I haven't used the move-left/move-right commands. It makes sense that that's a bit more complicated.

**Comment by gizmomogwai** [comment]
Hi @tdresser, please give version 0.6.0 a try. I tried to get the grasps of your elisp solution, but implemented it differently in 0.6.0. Interaction on tables with :compressed t is not possible anymore (for now, perhaps I will fix that in the future).

**Comment by tdresser** [comment]
Works for me, thanks a ton!
